parsl/executors/high_throughput/executor.py

```diff
@@ -186,15 +212,24 @@ def initialize_scaling(self):
         debug_opts = "--debug" if self.worker_debug else ""
         max_workers = "" if self.max_workers == float('inf') else "--max_workers={}".format(self.max_workers)
 
+        if self.interchange_address == "localhost":
```

Comment:
As an operator, I've found it frustrating in the past to have non-network related stuff change in behaviour depending on a specified hostname - postgres does it sometimes, for example.

What is this `if` actually really meant to be testing?